packages/vuetify/src/components/VGrid/_grid.sass

```diff
@@ -12,7 +12,7 @@
 
     > .row
       flex: 1 1 100%
-      max-width: 100%
+      max-width: calc(100% + #{$grid-gutter})
```

Comment:
This seems like it would cause a lot of issues if the user changes anything with padding in the container.

Reply:
v-row also has `dense` and `no-gutters` props.

Reply:
Thanks for review!

I've also tried to check `.container.fill-height > .row` with `dense` and `no-gutters`.

The top `.container` with red background is without `fill-height`, the second with blue is with `fill-height`, the third with green is to check #9731. 
It seems like the first and second have the same paddings.
My change affects only the second, so I think it's ok.

![image](https://user-images.githubusercontent.com/15768372/73454211-f3189100-43b0-11ea-97ed-c729992ce7cd.png)

### Markup:  

<details>

```vue
<template>
  <div style="width: 100%; height: 100%;">
    <div style="width: 100%; height: 35%">
      <v-container class="red">
        <v-row>
          <v-col v-for="n in 3" :key="n" cols="12" sm="4">
            <v-card class="pa-2" outlined tile>
              One of three columns
            </v-card>
          </v-col>
        </v-row>
        <v-row dense>
          <v-col v-for="n in 3" :key="n" cols="12" sm="4">
            <v-card class="pa-2" outlined tile>
              One of three columns (dense)
            </v-card>
          </v-col>
        </v-row>
        <v-row no-gutters>
          <v-col v-for="n in 3" :key="n" cols="12" sm="4">
            <v-card class="pa-2" outlined tile>
              One of three columns (no-gutters)
            </v-card>
          </v-col>
        </v-row>
      </v-container>
    </div>
    <div style="width: 100%; height: 35%">
      <v-container class="blue fill-height">
        <v-row>
          <v-col v-for="n in 3" :key="n" cols="12" sm="4">
            <v-card class="pa-2" outlined tile>
              One of three columns
            </v-card>
          </v-col>
        </v-row>
        <v-row dense>
          <v-col v-for="n in 3" :key="n" cols="12" sm="4">
            <v-card class="pa-2" outlined tile>
              One of three columns (dense)
            </v-card>
          </v-col>
        </v-row>
        <v-row no-gutters>
          <v-col v-for="n in 3" :key="n" cols="12" sm="4">
            <v-card class="pa-2" outlined tile>
              One of three columns (no-gutters)
            </v-card>
          </v-col>
        </v-row>
      </v-container>
    </div>
    <div style="width: 100%; height: 30%">
      <v-container class="green fill-height">
        <v-row justify="center">
          <v-col cols="4">
            <v-img :src="'http://placehold.it/2800x170'" height="100px"></v-img>
          </v-col>
        </v-row>
      </v-container>
    </div>
  </div>
</template>

<script>
export default {
  data: () => ({
    //
  })
};
</script>
```
</details>

(Sorry for my bad English.)